pkg/gemspec/parse.go

```diff
@@ -0,0 +1,104 @@
+package gemspec
+
+import (
+	"bufio"
+	"io"
+	"path/filepath"
+	"regexp"
+	"strings"
+	"unicode/utf8"
+
+	"github.com/aquasecurity/go-dep-parser/pkg/types"
+	"golang.org/x/xerrors"
+)
+
+var (
+	bwQuotes         = regexp.MustCompile(`['|"](.*?)['|"]`)
+	gemSpec          = ".gemspec"
+	gemStr           = ".name = "
```

Comment:
Spaces are mandatory? I wonder if `.name="foo"` is allowed.

Reply:
addressed in above comment